### DISCUSSION TITLE
What if someone gets my recovery phrase? Is it possible to change?

### DISCUSSION THREAD

**Discussion by siousu** [anyproto/contributors#19] [Q&A] [answered]
Hey all

maybe a stupid, silly question:

But let's assume someone has seen my recovery phrase because I've been careless and now they have access to all my data. (similar as if someone would have my password to another service).

Will it be possible to renew the recovery phrase in the future? I mean to change it like it is possible to change an API key or token? Or is this in the fundamental principle of the architecture not possible?

**Comment by Seanytype** [comment, marked as answer]
Hi there!

There's currently no way to change your Recovery Phrase, but we will implement a solution in the future. 

In the meantime, if your account is compromised you can follow these steps:

In-app top menu-
-  _Anytype > Space Settings > Export Data_ as a Protobuf file 
- _Account Settings > Data Management > Delete Account_
- _Profile > Log out_ 
- Create a new account (click _Join_ on start-up screen)
- _File > Import protobuf_

**Reply by siousu** [reply]
got it! Yes that would be the workaround atm. Sounds great, thank you!
